build.py

```diff
@@ -134,6 +147,30 @@ def main(*argv):
             "--cache-dir", cache_dir
         ])
 
+        if hash_names:
+            for fn in os.listdir(out_tmp_dir):
+                fn = os.path.join(out_tmp_dir, fn)
+                if os.path.isfile(fn):
+                    print(
+                        "Computing hashes for \"%s\"." % os.path.basename(fn)
+                    )
+
+                    fn_hashes = {}
+                    for hn in hash_names:
+                        h = subprocess.check_output(
+                            ["openssl", hn, fn],
+                            universal_newlines=True
+                        )
+                        h = h.split()[-1]
```

Comment:
As `openssl` likes to tack on a prefix (e.g. `SHA256(miniforge.sh)= `), this strips that part off and keeps the hash only.